src/Composer/Installers/LavaLiteInstaller.php

```diff
@@ -4,7 +4,7 @@
 class LavaLiteInstaller extends BaseInstaller
 {
     protected $locations = array(
-        'package' => 'packages/{$name}/',
+        'package' => 'packages/{$vendor}/{$name}/',
```

Comment:
Sorry, but can't break backward compatibility. You can create new locations or new installer.